src/lib/rest/orionReply.cpp

```diff
@@ -38,11 +38,9 @@ std::string orionReply(ConnectionInfo* ciP, const std::string& what, const std::
 {
    std::string out = "";
 
-   out += startTag1("",     "orion");
-   out += startTag1("  ",   "log");
-   out += valueTag1("    ",  what, value);
-   out += endTag("  ");
-   out += endTag("");
+   out += '{';
+   out += valueTag1(" ", what, value);
+   out += '}';
 
```

Comment:
While I like this change, it is not very backwards compatible ...
This needs to be clearly explained in the release notes for the next release.
However ... is it only used for `log` REST messages?